README.rst

```diff
@@ -306,6 +306,29 @@ Any valid ``pip`` flags or arguments may be passed on with ``pip-compile``'s
 Configuration
 -------------
 
+You can define project-level defaults for ``pip-compile`` and ``pip-sync`` by
+writing them to a configuration file in the same directory as your requirements
```

Comment:
Has this changed since the previous review? I think I saw CWD being used in a few places, not an input-related location...

Reply:
It had not, no. If you specify `src_files`, it looks in the directories of the `src_files` - if you do not specify `src_files`, it uses `cwd()`.

So `pip-compile /path/to/requirements.in` would look for `/path/to/.pip-tools.toml` or `/absolute/path/to/pyproject.toml` -- whereas `pip-compile` without arguments would look in the current working directory.

At first blush, that sounds like the behavior that would be expected, versus looking in the current directory for those files, even when compiling a source in another directory. Happy to revise if you disagree.

Reply:
I don't disagree, just wanted to clarify that I understand this correctly.

Reply:
I think this paragraph should mention the corner case of what's happening when the input is coming from stdin. Also, should we fall back to the output file directory when we read from stdin but the output is specified via a CLI option?

Reply:
Fair enough. Marking as resolved.